.travis.yml

```diff
@@ -14,17 +14,12 @@ sudo: false
 
 # Python versions to test
 python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
+  - "3.6"
 
 # Manually define here the combinations environment variables to test
 # https://github.com/travis-ci/travis-ci/issues/1519
 env:
```

Comment:
Let's move this to the travis environment as suggested by the FTL.

Reply:
Is it possible to set this in the environment?

Reply:
Do you mean the `env` command? It is set to the environment when Travis builds.